tests/code_layout/test_shellcheck.sh

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env bash
 
-# This runs shellcheck on all sh files
+# This runs shellcheck given or found sh files
```

Comment:
```suggestion
# This runs shellcheck on arguments files, or all files if none was given
```

Reply:
It's not "all files", is just ".sh" files if you don't pass a list.